### PR TITLE
CB-10487: Manually revert CB-7306, fix dfs replication values.

### PR DESCRIPTION
**Pull request by brycederriso**
Closes [CB-10487](https://jira.cloudera.com/browse/CB-10487)

* Remove code that added dfs_replication to hdfs service configs
* Add `dfs_replication` values to templates:
  * 7.2.0
  * 7.2.1
  * 7.2.2
  * 7.2.6
  * 7.2.7
  * 7.2.7 HA
  * 7.2.8
  * 7.2.8 HA

Tested by setting up Data Lakes using `cbd` tool locally.

### CODE REVIEW

File: core/src/main/resources/defaults/blueprints/7.2.1/cdp-sdx.bp [hunk 1]
```diff
@@ -280,7 +280,11 @@
             {
               "name": "service_config_suppression_datanode_count_validator",
               "value": "true"
-            }
+            },
+            {
+              "name": "dfs_replication",
+              "value": "1"
+            },
```
Comment (orngejaket) [inline core/src/main/resources/defaults/blueprints/7.2.1/cdp-sdx.bp:287]: remove extra ,

**Review by orngejaket — CHANGES_REQUESTED** [review]

**Comment by orngejaket** [comment]
LGTM other than extra comma

**Comment by brycederriso** [comment]
Tested 7.2.7 Light duty replication value on CBD:
7 2 7 light duty replication

**Comment by brycederriso** [comment]
Tested 7.2.7 Medium Duty replication value on CBD:
7 2 7 medium duty replication

**Comment by brycederriso** [comment]
Also, tested successful provisioning of:
* 7.2.0
* 7.2.1
* 7.2.2
* 7.2.6

**Review by orngejaket — APPROVED** [review]

**Pull request merged**